setup.py

```diff
@@ -85,8 +115,11 @@ class BuildExt(build_ext):
         'unix': [],
     }
 
+    UnixCCompiler.src_extensions.append(".mm")
+    UnixCCompiler.language_map[".mm"] = "objc"
+
     if sys.platform == 'darwin':
-        darwin_opts = ['-stdlib=libc++', '-mmacosx-version-min=10.7']
```

Comment:
Not sure why this was here, I removed it while debugging some bug and never added it back

Reply:
It's used for building native extensions (or more relevant, binary wheels) on a macOS version that's newer than the target version to be supported, which can come in handy in CI environments where access to older versions is not always easy. It's fine to leave this flag out in `setup.py` though as it makes more sense to define it from the outside using the `MACOSX_DEPLOYMENT_TARGET` environment variable. I do the same here: https://github.com/letmaik/rawpy/blob/master/.github/scripts/build-macos.sh

Reply:
Okay. Keeping as-is.